internal/engine/experiment/stunreachability/stunreachability_internal_test.go

```diff
@@ -1,18 +0,0 @@
-package stunreachability
-
-import (
-	"context"
-	"net"
-
-	"github.com/pion/stun"
-)
-
-func (c *Config) SetNewClient(
-	f func(conn stun.Connection, options ...stun.ClientOption) (*stun.Client, error)) {
-	c.newClient = f
-}
-
-func (c *Config) SetDialContext(
-	f func(ctx context.Context, network, address string) (net.Conn, error)) {
-	c.dialContext = f
-}
```

Comment:
These functions were here because I used external testing. I choose to convert to internal testing to simplify testing, so they are gone now.